data/Pate_1990/metadata.yml

```diff
@@ -0,0 +1,321 @@
+          )
+        )
+    '
+  collection_date: 1987/1989
+  taxon_name: taxon_name
+  location_name: location
+  description: A study examining the growth and storage traits of seedlings growing
+    after recent fires in fire-prone, Mediterranean-type ecosystems in south-western
+    Australia.
+  basis_of_record: field
+  life_stage: seedling
+  sampling_strategy: Study sites and plant species ^p Seedlings of a range of woody
+    and herbaceous dicotyledonous species were collected during the period December
+    1987-January 1989 from four sites which had been subjected to fire 2-4 years previously.
+    (1) Coastal heath shrublands on deep calcareous or siliceous sands at Parry's
+    Inlet, Denmark (34°58'S, 117°21'E). (2) Understorey of jarrah (Eucalyptus marginata)
+    forest on gravelly lateritic soil at Kelmscott (32°07'S, 116°01'E). (3) Banksia
+    woodlands (B. attenuata, B. menziesii, B. ilicifolia and B. prionotes) on deep
+    leached sands at Yanchep (31°31'S, 115°41'E). (4) Banksia shrublands (B. hookeriana,
+    B. attentuata, B. menziesii) on deep, heavily leached sands at Eneabba (29°49'S,
+    115°61'E). The taxa investigated totalled 66 species, 27 seeders, 37 resprouters
+    and two species initially classified as facultative seeder/resprouters. One of
+    the latter (Acacia extensa) typically failed to survive fire in the habitats examined
+    in this study, and so was grouped with the seeders, while the ecotype studied
+    of the second species (Conospermum triplinervium) was classified for present purposes
+    as a resprouter species in showing generally high resistance to fire at its study
+    site. Elsewhere this species has been regarded as a seeder (Dixon, 1987). The
+    species chosen included representatives of 17 families of dicotyledons, but were
+    predominantly species of three species-rich families Fabaceae, Proteaceae and
+    Myrtaceae.
+  original_file: none; transcribed from manuscript
+  notes: transcribed from manuscript
+locations:
+  Mediterranean-type ecosystems in south-western Australia:
+    latitude (deg): .na
+    longitude (deg): .na
+    latitude range (deg): -29.82 to -34.97
+    longitude range (deg): 115.68 to 117.35
+    description: (1) Coastal heath shrublands on deep calcareous or siliceous sands
+      at Parry's Inlet, Denmark (34°58'S, 117°21'E). (2) Understorey of jarrah (Eucalyptus
+      marginata) forest on gravelly lateritic soil at Kelmscott (32°07'S, 116°01'E).
+      (3) Banksia woodlands (B. attenuata, B. menziesii, B. ilicifolia and B. prionotes)
+      on deep leached sands at Yanchep (31°31'S, 115°41'E). (4) Banksia shrublands
+      (B. hookeriana, B. attentuata, B. menziesii) on deep, heavily leached sands
+      at Eneabba (29°49'S, 115°61'E).
+    fire history (years since fire): 2-4
+contexts:
+- context_property: plant age
+  category: entity_context
+  var_in: plant_age
+  values:
+  - value: 2.5
```

Comment:
The description shouldn't just be used to document units, because it is the same for every value, which doesn't make sense. Instead include the units in the context property, for instance `plant age (years)` and `total dry mass (g)`. We do this elsewhere, austraits$contexts %>% distinct(context_property)

And then you can just leave out the description rows.